lib/ansible/modules/remote_management/redfish/idrac_redfish_config.py

```diff
@@ -213,9 +241,17 @@ def main():
             if command == "SetManagerAttributes":
                 result = rf_utils.set_manager_attributes(mgr_attributes)
 
+    if any((module.params['manager_attribute_name'], module.params['manager_attribute_value'])):
+        result['warnings'] = [('Arguments `manager_attribute_name` and `manager_attribute_value` '
+                               'are deprecated since Ansible 2.9. Use '
+                               '`manager_attributes` instead for passing in the '
+                               'manager attribute name and value pairs.')]
```

Comment:
Probably better to use the module.deprecate() method here. For example, see:
https://github.com/ansible/ansible/blob/61456313025e29fd4656654853e3468a928a6ca0/lib/ansible/modules/remote_management/redfish/redfish_config.py#L238

Reply:
fixed